src/platform/term.c

```diff
@@ -887,6 +890,25 @@ int term_keycodeByName(const char *name) {
     return name[0];
 }
 
+static int term_ctrlPressed(int* key) {
+    // TODO: are there other keys that need to be handled here?
+    if (*key == 13) {
+        *key = RETURN_KEY;
+        return 0;
+    }
+    if (*key == 27 || *key == KEY_BACKSPACE
+        || *key == KEY_LEFT || *key == KEY_RIGHT || *key == KEY_UP || *key == KEY_DOWN) {
+        return 0;
```

Comment:
Are these branches needed because _unctrl_ fails on some keys? Is it possible to use another ncurses function, maybe *keyname*?

Reply:
It looks like `keyname` works better here; for instance, `unctrl` returns "^C" for the up arrow, while `keyname` returns `KEY_UP`. The only special case that's still needed is for newlines, I think.